docs/about/release-notes.rst

```diff
@@ -51,6 +51,7 @@ Breaking changes
 
 * Coordinates are no longer converted to attributes when slicing or by ``transform_coords``.
   Instead, they become unaligned `#3153 <https://github.com/scipp/scipp/pull/3153>`_.
+* By default, ``sc.compat.from_pandas`` now returns data with ``unit=None`` (used to be dimensionless) `#3169 <https://github.com/scipp/scipp/pull/3169>`_.
```

Comment:
I don't feel this is a useful change. What is your exact argument?

Reply:
To indicate that the unit is unknown. Isn't this what we do in scippnexus?

Reply:
NeXus has support for units, so when no unit is given then `None` makes sense. Pandas does not support units. If so, are you suggesting we should also change `sc.array`?

Reply:
`sc.array` is more explicit. You either pass a unit or get the default. `from_pandas` handles units automatically.

Are you worried that this decreases usability?

Reply:
Yes, I think it would become pretty useless.

Reply:
> from_pandas handles units automatically.

How?

Reply:
> > from_pandas handles units automatically.
> 
> How?

With the parser. This is not as explicit as in `array`. And I'd expect 'no parser' to result in 'no unit'.

Reply:
But encoding units in the header is a kind of special case. I don't think this is a good argument for making `from_pandas` useless for others?

Reply:
Obviously forcing users to always specify units would be ideal, but we do not do that for any of our other creation methods either --- you could argue that `dimensionless` isn't a good default for `linspace`, `array`, ...

So I would favor keeping it useful and consistent? Unless you advocate for changing all the other creation methods as well, making the `unit` argument mandatory (but if you do, then this should not be part of the PR)?

Reply:
Ok. Changed it back. But i kept `unit=None` for the row coord as that is an index (a la `sc.index`).

Reply:
I would say that dimensionless is better in this case :+1:

Reply:
> Ok. Changed it back. But i kept `unit=None` for the row coord as that is an index (a la `sc.index`).

But it often is not! You can have floats and other things indexing rows in Pandas.

Reply:
Hmm, I think this was reverted already? Can you mark the conversation as resolved if it was?